Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector.yml

```diff
@@ -0,0 +1,156 @@
+category: Analytics & SIEM
+commonfields:
+  id: SaasSecurityEventCollector
+  version: -1
+configuration:
+- additionalinfo: 'The instance configuration URL based on the server location: <br/>US:
+    https://api.aperture.paloaltonetworks.com<br/> EU: https://api.aperture-eu.paloaltonetworks.com<br/>
+    APAC: https://api.aperture-apac.paloaltonetworks.com'
+  defaultvalue: https://api.aperture.paloaltonetworks.com
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- additionalinfo: The SaaS Security Client ID and Client Secret.
+  display: Client ID
+  displaypassword: Client Secret
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: The maximum number of events per fetch.
+  additionalinfo: How many events to fetch every time fetch is being executed. Default is 100. Must be divisible by 100 due to Saas-Security api limitations.
+  defaultvalue: 100
+  name: max_fetch
+  required: true
+  type: 0
+description: Palo Alto Saas Security Event Collector integration for XSIAM.
+display: Saas Security Event Collector
+name: SaasSecurityEventCollector
+script:
+  commands:
+  - deprecated: false
+    description: Manual command to fetch events and display them.
+    execution: false
+    name: saas-security-get-events
+    arguments:
+      - name: limit
+        description: The maximum number of events to get. Must be divisible by 100 due to Saas-Security api limitations
+        defaultValue: 100
+      - auto: PREDEFINED
+        defaultValue: 'False'
+        description: Set this argument to True in order to create events, otherwise the command will only display them. If setting to 'False' The returned events will be lost.
+        isArray: false
+        name: should_push_events
+        predefined:
+          - 'True'
+          - 'False'
+        required: true
+    outputs:
+    - contextPath: SaasSecurity.Event.log_type
+      description: Event type.
+      type: String
+    - contextPath: SaasSecurity.Event.item_type
+      description: Item type (File, Folder, or User).
+      type: String
+    - contextPath: SaasSecurity.Event.item_name
+      description: Name of the file, folder, or user associated with the event.
+      type: String
+    - contextPath: SaasSecurity.Event.item_unique_id
+      description: Unique ID number for an asset’s related asset.
```

Comment:
asset's related asset? Is that right?

Reply:
this was copied directly from Saas-Security documentation.

https://docs.paloaltonetworks.com/saas-security/saas-security-admin/saas-security-api/syslog-and-api-integration/api-client-integration/public-api-references/log-events-api

![image](https://user-images.githubusercontent.com/53861351/171105948-aa99ee8c-4277-46ff-8cd8-758ace927793.png)